Riot/AppDelegate.m

```diff
@@ -1710,20 +1733,21 @@ - (void)addMatrixCallObserver
         [[NSNotificationCenter defaultCenter] removeObserver:matrixCallObserver];
     }
     
-    // Register call observer in order to handle new opened session
-    matrixCallObserver = [[NSNotificationCenter defaultCenter] addObserverForName:kMXCallManagerNewCall object:nil queue:[NSOperationQueue mainQueue] usingBlock:^(NSNotification *notif)
-    {
-        
+    void (^handleNotification)(NSNotification *) = ^(NSNotification *notif) {
```

Comment:
I guess you are disturbed by the crazy amount of indentation spaces automatically added by Xcode.
I agree it is boring. You can add a line break before the ```{``` of the block. That will make the default editor configuration of xcode indents normally.

For future maintenance, please keep inline block code. Else it feels we kill the benefit of GCD.